packages/venia-concept/src/reducers/cart.js

```diff
@@ -9,7 +9,9 @@ export const initialState = {
     details: {},
     guestCartId: null,
     shippingMethods: [],
-    totals: {}
+    totals: {},
+    addToCartActiveRequest: false,
+    recentAddItemHadError: false
```

Comment:
Why do we store boolean variable here? Is it better to store error string here?

Reply:
We don't have to use string description anywhere, so it's useless

Reply:
In this case it is better to change the name of the property in state to show, that it has a boolean value.

Reply:
But I still think that it is better to store error message here, I suppose we need to display it in the future.